### PR TITLE
chore: separate API docs from main site

### PR DESCRIPTION
**Pull request by mujahidkay**
Closes: https://github.com/Agoric/documentation/issues/1187

This PR aims to remove agoric-sdk and endo reference docs from docs.agoric.com, adds sidebar corresponding sidebar entries and redirects them to the relevant pages.

### CODE REVIEW

File: main/.vitepress/themeConfig/nav.js [hunk 1]
```diff
@@ -38,6 +38,14 @@ export const nav = [
         ariaLabel: 'Agoric CLI',
         link: '/guides/agoric-cli/',
       },
+      {
+        text: 'Agoric API Reference',
```
Comment (mujahidkay) [inline main/.vitepress/themeConfig/nav.js:42]: This entry seems apt to me but open to suggestion if we wanna call it something else

Comment (dckc) [inline main/.vitepress/themeConfig/nav.js:42]: I think we usually include SDK in the name for this stuff:

```suggestion
        text: 'Agoric SDK API Reference',
```

File: main/.vitepress/config.mjs [hunk 2]
```diff
@@ -229,6 +229,14 @@ export default defineConfig({
                 },
               ],
             },
+            {
+                text: 'Agoric API Reference',
```
Comment (dckc) [inline main/.vitepress/config.mjs:233]:
```suggestion
                text: 'Agoric SDK API Reference',
```

**Review by dckc — APPROVED** [review]
one suggestion

**Comment by dckc** [comment]
I presume it's OK for me to go ahead and merge this. I keep tripping over the lack of it.

**Pull request merged**